index.bs

```diff
@@ -443,9 +445,10 @@ spec:dom-ls; type:interface; text:Document
 
   <pre dfn-type="grammar" link-type="grammar">
       <dfn>suborigin-name</dfn> = 1*( <a>LOWERALPHA</a> / <a>DIGIT</a> / "-" )
-      <dfn>suborigin-policy-option</dfn> = "'unsafe-postmessage-send'"
-                                / "'unsafe-postmessage-receive'"
+      <dfn>suborigin-policy-option</dfn> = "'force-include-credentials'"
```

Comment:
It doesn't seem quite as unsafe as the other options, but I suppose you're right. I'll change it to 'unsafe-include-credentials' (don't want it to be too long of a name either)